Makefile

```diff
@@ -487,3 +487,16 @@ dev-release:
 validate-release: setup-venv
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install yq==2.10.0
 	BUNDLE_VERSION=$(BASE_BUNDLE_VERSION) CHANNEL="beta" ./hack/validate-release.sh
+
+# Operator-sdk release version
+RELEASE_VERSION ?= v0.16.0
```

Comment:
please move this var at the top of Makefile, where other vars are - it would make the maintenance easier. Also, please name it like `OPERATOR_SDK_VERSION`